pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md

```diff
@@ -94,22 +94,39 @@ In the fourth step of creation, we attach it to the private network we created b
 
 ![Setting-up PCI - Attaching to a private network](images/vrack-example-05.png){.thumbnail}
 
-After instance creation, we can see the connection details in the OVHcloud Control Panel. If we log in to the instance using SSH, we can see that it has two network interfaces, one attached to the public IP address we use to log in, the other attached to the private network:
+After instance creation, click on the name of the new created instance, and then scroll down a little in order to see the connection details in the OVHcloud Control Panel.
```

Comment:
"click on the name of the new created instance" -> "click on the name of the newly created instance"

Reply:
fixed